bindata/network/ovn-kubernetes/ovnkube-master.yaml

```diff
@@ -322,7 +323,8 @@ spec:
                   if ovsdb-tool compare-versions "$db_version" == "$target_version"; then
                     :
                   elif ovsdb-tool compare-versions "$db_version" ">" "$target_version"; then
-                      echo "Database $schema_name has newer schema version ($db_version) than our local schema ($target_version), possibly an upgrade is partially complete?"
+                      echo "Database $schema_name has newer schema version ($db_version) than our local schema ($target_version), destroy the DB"
+                      rm -rf /etc/ovn/ovnsb_db.db
```

Comment:
Also I'm not sure putting this 2nd downgrade commit into 4.4 is right. The 4.5 and 4.6 commits for fixing upgrades do not have this commit right? Maybe it would be better to just stick to the original commit and get that in with this PR, file another bug about possible db downgrade case and get that commit into 4.6,4.5,4.4? Or is this a one time thing we only need in 4.4?

Reply:
Yeah, it was more for consistency between version + having one less PR to do. But yes, we will never downgrade on 4.4. I will remove it.